server/src/main/java/org/elasticsearch/action/search/SearchRequestBuilder.java

```diff
@@ -61,15 +61,6 @@ public SearchRequestBuilder setSearchType(SearchType searchType) {
         return this;
     }
 
-    /**
-     * The a string representation search type to execute, defaults to {@link SearchType#DEFAULT}. Can be
-     * one of "dfs_query_then_fetch" or "query_then_fetch".
-     */
-    public SearchRequestBuilder setSearchType(String searchType) {
```

Comment:
Can we be sure this isn't called somewhere outside our project (i.e. clients etc...)? Maybe deprecate it first if not.

Reply:
There's no more client code like that :) We are not releasing the transport client any longer so we can do whatever we'd like here. I'd really like to do away with all this unused stuff eventually it's thousands of lines :)

Reply:
Fine by me, just wanted to point this out and make sure we're okay with this.